timemachine/cpp/src/nonbonded_interaction_group.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include "neighborlist.hpp"
+#include "potential.hpp"
+#include "vendored/jitify.hpp"
+#include <array>
+#include <set>
+#include <vector>
+
+namespace timemachine {
+
+typedef void (*k_nonbonded_fn)(
```

Comment:
Do we need multiple copies of this typdef?

Reply:
Added to #639